docs/bfb.md

```diff
@@ -97,22 +97,52 @@ MQTT Topic: selfupdate/desiredstatefeedback
 }
 ```
 
+## Command for single self-update action/step
+This message contains information about SUA action - what must be done as a single step - download, update, activate, cleanup.
+MQTT Topic: selfupdate/desiredstate/command
+```
+{
+    "activityId": "random-uuid-as-string",
+    "timestamp": 123456789,
+    "payload": {
+        "baseline": "BASELINE NAME",
+        "command": "command_type"
```

Comment:
"command_type" contains underscore, "BASELINE NAME" not - why?
Is uppercase / lowercase important?

Reply:
This is a placeholder. In future will be replaced with base line name. Just a string.